resources/language/en.ini

```diff
@@ -146,8 +175,8 @@ label.bug = "Bug"
 label.ticket_title = "To-Do Name"
 label.milestone = "Milestone"
 label.not_assigned_to_milestone = "Not assigned to a milestone"
-label.todo_type = "Type"
```

Comment:
I don't see the replacement in dashboard and tickets. Please be careful renaming labels since they can be used in other templates.

Reply:
I re-added these as they were to be safe. There will be a duplicate value, but you're right. I wasn't thinking about elsewhere at the moment.